dashboard/src/index.css

```
@@ -1,120 +1,259 @@
+@import url('https://fonts.googleapis.com/css2?family=Kalam:wght@400;700&family=Patrick+Hand&display=swap');
+
 @tailwind base;
```

Biome (2.3.13)

[error] Unexpected unknown at-rule: tailwind 

tailwind is not a standard CSS at-rule, which may lead to unexpected styling results or failure to interpret the styles as intended.
See MDN web docs for a known list of at-rules.
To fix this issue, consider removing the unknown at-rule.

(lint/suspicious/noUnknownAtRules)